### ISSUE TITLE
Storage Commands tutorial bug(?)

### ISSUE THREAD

**Issue by smrgit** [GoogleCloudPlatform/datalab#709]
In the Storage Commands tutorial the %storage write command is used like this:
```
%%storage write text $sample_bucket_object
```
but when I tried doing something similar I had to do this to get it to work:
```
%storage  write -v outFile -o $cohorts_bucket_object
```

If I try the command the way it is in the tutorial (with either % or %%), I get this:
```
storage write: error: argument -v/--variable is required

usage: storage write [-h] -v VARIABLE -o OBJECT

optional arguments:
  -h, --help            show this help message and exit
  -v VARIABLE, --variable VARIABLE
                        The name of the source Python variable
  -o OBJECT, --object OBJECT
                        The name of the destination GCS object to write
```

**Comment by gramster** [comment]
Thanks for reporting! I have a notebook test tool and this notebook was inadvertently excluded so it is out of date.

**Comment by gramster** [comment]
Fixed in c13839590e6431f1189a738378353592139e556d
